Podfile

```diff
@@ -48,12 +50,12 @@ pod 'ARAnalytics/HockeyApp'
 
 pod 'CardFlight'
 pod 'ECPhoneNumberFormatter'
-pod 'UIImageViewAligned', :git => "https://github.com/orta/UIImageViewAligned.git"
+pod 'UIImageViewAligned', :git => "https://github.com/ashfurrow/UIImageViewAligned.git"
```

Comment:
Only thing I changed was to add a tag on my fork ( https://github.com/ashfurrow/UIImageViewAligned/commit/6f8cb5669b219ef08c90c2e2e5a082cadfe7565b ) – if it's on trunk, then perfect :+1: